### PR TITLE
feat: support the namespace offloadThresholdInSeconds API in pulsaradmin pkg

### PR DESCRIPTION
**Pull request by ericsyh**
*(If this PR fixes a github issue, please add `Fixes #<xyz>`.)*

Fixes #<xyz>

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: #<xyz>

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API: (yes / no)
  - The schema: (yes / no / don't know)
  - The default values of configurations: (yes / no)
  - The wire protocol: (yes / no)

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / GoDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: pulsaradmin/pkg/admin/namespace.go [hunk 1]
```diff
@@ -114,6 +114,12 @@ type Namespaces interface {
 	// GetOffloadThreshold returns the offloadThreshold for a namespace
 	GetOffloadThreshold(namespace utils.NameSpaceName) (int64, error)
 
+	// SetOffloadThreshold sets the offloadThresholdInSeconds for a namespace
```
Comment (RobertIndie) [inline pulsaradmin/pkg/admin/namespace.go:117]:
```suggestion
	// SetOffloadThresholdInSeconds sets the offloadThresholdInSeconds for a namespace
```

File: pulsaradmin/pkg/admin/namespace.go [hunk 2]
```diff
@@ -114,6 +114,12 @@ type Namespaces interface {
...
+	SetOffloadThresholdInSeconds(namespace utils.NameSpaceName, threshold int64) error
+
+	// GetOffloadThreshold returns the offloadThresholdInSeconds for a namespace
```
Comment (RobertIndie) [inline pulsaradmin/pkg/admin/namespace.go:120]:
```suggestion
	// GetOffloadThresholdInSeconds returns the offloadThresholdInSeconds for a namespace
```

**Comment by ericsyh** [comment]
Okay, I will add test for new funcs.

File: pulsaradmin/pkg/admin/namespace_test.go [hunk 3]
```diff
@@ -277,3 +277,67 @@ func TestNamespaces_GetSubscriptionExpirationTime(t *testing.T) {
...
+	namespace, _ := utils.GetNamespaceName("public/default")
+
+	// set the subscription expiration time and get it
+	err = admin.Namespaces().SetOffloadThresholdInSeconds(*namespace,
+		60)
+	assert.Equal(t, nil, err)
+	offloadThresholdInSeconds, err := admin.Namespaces().GetOffloadThresholdInSeconds(*namespace)
+	assert.Equal(t, nil, err)
+	expected := 60
+	assert.Equal(t, expected, offloadThresholdInSeconds)
```
Comment (crossoverJie) [inline pulsaradmin/pkg/admin/namespace_test.go:342]: image

Please fix the exception here.

Comment (ericsyh) [reply pulsaradmin/pkg/admin/namespace_test.go:342]: fixed in the latest commit.

**Pull request merged**